modules/dnn/test/test_layers.cpp

```diff
@@ -777,6 +777,125 @@ TEST(Layer_PriorBox, squares)
     normAssert(out.reshape(1, 4), target);
 }
 
+TEST(Layer_Test_DWconv_Prelu, Accuracy)
+{
+    // Test case
+    // input       img size 3x16x16  value all 1
+    //   |
+    //   v
+    // dw_conv     weight[0]=-1 weight[1]=-2 weight[2]=-3   bias={1,2,3}
+    //   |
+    //   v
+    // prelu       weight={1,2,3}
+    //   |
+    //   v
+    // output      out size 3x14x14  if right: out[0]=-8 out[0]=-32 out[0]=-72
+    //             but current opencv output: out[0]=-24 out[0]=-48 out[0]=-72
+
+    const int num_input = 6;    //inpChannels
+    const int group = 3;        //outChannels=group when group>1
+    int num_output = 6;         //only need to set when group=1
+
+    num_output = group == 1 ? num_output : group;   //conv or group conv
```

Comment:
Please do not overcomplicate your test. I just asked you to check a case with number of input channels == 3, number of output channels == 6 and a group number 3 because it works in a wrong way for now.

Reply:
sorry